src/tools/updateLegacyPopulations.js

```diff
@@ -0,0 +1,57 @@
+import { Op } from 'sequelize';
+import { ActivityReport } from '../models';
+import { auditLogger } from '../logger';
+
+const newPopulations = {
+  'Infant/Toddlers': 'Infants and Toddlers (ages birth to 3)',
+  Preschool: 'Preschool (ages 3-5)',
```

Comment:
should Preschool be in quotes?

Reply:
Doesn't need to be (also, the linter won't allow it)